app/jobs/update_pharmgkb.rb

```diff
@@ -0,0 +1,50 @@
+require 'net/http'
+require 'openssl'
+require 'zip'
+
+OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
+
+class UpdatePharmgkb < TsvUpdater
+  def tempfile_name
+    ['interactions', '.tsv']
```

Comment:
I would suggest a unique name here that is not also used by any of the other app/job TsvUpdaters. This is just in case the two would ever run at the same time so that one updater doesn't overwrite the tempfile of another updater.

Reply:
changed to `pharmgkb_interactions.tsv`